### PR TITLE
Pool Upgrade docs

### PR DESCRIPTION
**Pull request by ashcherbakov**
Signed-off-by: ashcherbakov <alexander.sherbakov@dsr-company.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,15 +161,23 @@ If you made changes in both indy-plenum and indy-node, you need to do the follow
 
 ## Docs and links
 
-- Please have a look at documents ad diagrams in [docs](docs) folder
 - Indy-plenum is based on [RBFT](https://pakupaku.me/plaublin/rbft/5000a297.pdf) protocol
+- Please have a look at documents ad diagrams in [docs](docs) folder
```
Comment (andkononykhin) [inline README.md]: documents and diagrams

File: README.md [hunk 2]
```diff
@@ -157,15 +161,23 @@ If you made changes in both indy-plenum and indy-node, you need to do the follow
...
+- Please have a look at documents ad diagrams in Plenum's [docs](https://github.com/hyperledger/indy-plenum/tree/master/docs) folder:
```
Comment (andkononykhin) [inline README.md]: the same as above

File: docs/pool-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- Each node sends `NODE_UPGRADE` transaction twice:
+    - `in_progress` action: just before start of the Upgrade (that is re-starting the node and applying a new package)
+    to log that Upgrade started on the node.
+    - `success` or `fail` action: after upgrade of the node to log the upgrade result.
+- `NODE_UPGRADE` transaction is a common transaction (written to config ledger), so consensus is required.
+
+### Node Control Tool
+
+- Upgrade is performed by a `node-control-tool`.
+- See `node_control_tool.py`.
```
Comment (andkononykhin) [inline docs/pool-upgrade.md]: I think a link will be helpful here

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

**Comment by andkononykhin** [comment]
test this please

**Pull request merged**